library/linux/Makefile

```diff
@@ -50,7 +60,3 @@ $(FLUTTER_ENGINE_LIB_FILE): $(FLUTTER_DIR)/bin/internal/engine.version
 	if [ -f $(FLUTTER_ENGINE_HEADER) ]; then \
 		mv $(FLUTTER_ENGINE_HEADER) include/; \
 		fi
-
-.PHONY: clean
-clean:
```

Comment:
This seems highly subjective to me; having `clean` at the end seems entirely reasonable. Is this based on a style guide of some kind, or just personal preference?

Reply:
I think abstract targets must be together, not to be dispersed throughout the Makefile. This why I put `clean` and `all` together.